Connectors/Rhino/Speckle.Connectors.GrasshopperShared/Components/Dev/DeconstructSpeckleParam.cs

```diff
@@ -47,9 +47,21 @@ protected override void SolveInstance(IGH_DataAccess da)
 
     switch (data)
     {
-      case SpeckleWrapper wrapper:
-        Name = string.IsNullOrEmpty(wrapper.Name) ? wrapper.Base.speckle_type : wrapper.Name;
-        outputParams = CreateOutputParamsFromBase(wrapper.Base);
+      case SpeckleObjectWrapperGoo objectGoo when objectGoo.Value != null:
```

Comment:
why was this change necessary? shouldn't all wrapper classes be handled the same?

Reply:
Hmm I'm still not seeing why the original code no longer works - let's sync on this quickly when you're back

Reply:
@bjoernsteinhagen I've updated this component to preserve the original funcitonality - tested with materials and collections, not yet with block instances and defs (but should work as before)